lib/commands/general.js

```diff
@@ -124,11 +124,12 @@ commands.background = async function (seconds) {
   if (this.opts.startActivityArgs && this.opts.startActivityArgs[`${appPackage}/${appActivity}`]) {
     // the activity was started with `startActivity`, so use those args to restart
     args = this.opts.startActivityArgs[`${appPackage}/${appActivity}`];
-  } else if (appPackage === this.opts.appPackage && appActivity === this.opts.appActivity) {
+  } else if (appPackage === this.opts.appPackage && appActivity === this.opts.appActivity ||
```

Comment:
i think it would be good to add parentheses for grouping to make it visually clear the boolean operator precedence